backend/server/tasks.py

```diff
@@ -6,10 +6,11 @@
 import backend.server.utils.deploy as deploy
 import backend.server.utils.github as github
 import backend.settings as settings
-from backend.server.models import Project, Wallet, BlockchainState
+from backend.server.models import Project, Wallet, BlockchainState, ASC
 from backend.server.utils.contract_compiler import ContractCompiler
 
 REDIS_CLIENT = StrictRedis.from_url(settings.CELERY_BROKER_URL)
+REDIS_LOCK_NAME = 'create-contract-lock'
```

Comment:
can we get a better REDIS lock name? Something like REDIS_CREATE_CONTRACT_LOCK?

Reply:
done